pkgs/tools/X11/devour/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, libX11 }:
+stdenv.mkDerivation rec {
+  version = "12";
+  name = "devour-${version}";
+  src = fetchFromGitHub {
+    owner = "salman-abedin";
+    repo = "devour";
+    rev = version;
+    sha256 = "1qq5l6d0fn8azg7sj7a4m2jsmhlpswl5793clcxs1p34vy4wb2lp";
+  };
+
+  installPhase = ''
+    mkdir -p $out/local/usr/bin
+    mv devour $out/local/usr/bin
+  '';
+
+  buildInputs = [ libX11 ];
+
+  meta = with stdenv.lib; {
+    description = "Devour hides your current window before launching an external program and unhides it after quitting.";
+    homepage = "https://github.com/salman-abedin/devour";
+    maintainers = [ maintainers.mazurel ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ mazurel ];
```

Reply:
Why is this a problem? The full maintainer list doesn't need to be in scope when there's just one entry, no? Looks a lot cleaner the way it was before IMO